### ISSUE TITLE
CMake won't genenerate file sqlite3mc.pc 

### ISSUE THREAD

**Issue by taozuhong** [utelle/SQLite3MultipleCiphers#131]
The `configure/make` build script will output file `sqlite3mc.pc`,

but not yet cmake build script

**Comment by taozuhong** [comment]
Here is an example to generate sqlite3mc.pc with CMake:
https://discourse.cmake.org/t/how-to-generate-pc-pkg-config-file-supporting-prefix-of-the-cmake-install/4109

**Comment by utelle** [comment]
> Here is an example to generate sqlite3mc.pc with CMake: https://discourse.cmake.org/t/how-to-generate-pc-pkg-config-file-supporting-prefix-of-the-cmake-install/4109

Nice reading ... but it seems to be rather complex. I'm not a CMake expert. Therefore it would be nice if someone with good CMake knowledge would tackle this task.

**Comment by jammerxd** [comment]
@taozuhong I'll look into this if I get some time - did you need something specific from the .pc file? Or why is it required?

**Comment by utelle** [comment]
> Maybe it's good to keep the pc file same as makefile's output pc.

Yes, the resulting file `sqlite3mc.pc` should be identical to the one created by the autoconf/automake tool chain, if possible.
